src/utils/validationSchema.js

```diff
@@ -0,0 +1,43 @@
+import Joi from 'joi';
+/**
+ * This is the schema definition
+ * for a new userAccount
+ */
+const newUserSchema = Joi.object().keys({
+  fullName: Joi
+    .string()
+    .trim(),
+  username: Joi
+    .string()
+    .alphanum()
+    .lowercase()
+    .trim()
+    .min(8)
+    .required(),
+  email: Joi
+    .string()
+    .lowercase()
+    .trim()
+    .email({ minDomainAtoms: 2 })
+    .required(),
+  password: Joi
+    .string()
+    .alphanum()
+    .min(8)
+    .required(),
+  passwordConfirmation: Joi
+    .string()
+    .required()
+    .valid(Joi.ref('password'))
+    .options({
+      language: {
+        any: {
+          allowOnly: 'Passwords do not match!!',
+        }
+      }
+    })
+});
+
+export default {
```

Comment:
instead of doing this, you can just export the object directly using 
`export const newUserSchema = Joi.object.....`

Reply:
Okay. But eslint flagged it that it prefers export default when it is just one single costant.  http://airbnb.io/javascript/#modules--prefer-default-export

Reply:
@nwamugo just reminded me that we can add a rule to override this behaviour in the eslint config file. Should i go ahead and implement that?